pkg/distributor/forwarding/forwarding_test.go

```diff
@@ -770,6 +775,161 @@ func BenchmarkRemoteWriteForwarding(b *testing.B) {
 	}
 }
 
+func TestForwardingToHttpgrpcTarget(t *testing.T) {
```

Comment:
In other places we treated `http` and `grpc` as two separate terms
```suggestion
func TestForwardingToHttpGrpcTarget(t *testing.T) {
```